finalPrep/modZeroHero.js

```diff
@@ -56,16 +78,24 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "King Kong",
+  smell: "smelly",
+  weight: 3500,
+  citiesDestroyed: ["new york city", "hong kong"],
+  luckyNumbers: [8,5,99],
+  address: {number: 222, street: "olympic ave", state: "NY", zip:"20021"}
```

Comment:
I encourage giving each key/value pair their own line. Just be sure to indent extra on those lines!